crates/notary/server/README.md

```diff
@@ -1,109 +1,141 @@
+  </head>
+  <body>
+    <svg width="86" height="88" viewBox="0 0 86 88" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M25.5484 0.708986C25.5484 0.17436 26.1196 -0.167376 26.5923 0.0844205L33.6891 3.86446C33.9202 3.98756 34.0645 4.22766 34.0645 4.48902V9.44049H37.6129C38.0048 9.44049 38.3226 9.75747 38.3226 10.1485V21.4766L36.1936 20.0606V11.5645H34.0645V80.9919C34.0645 81.1134 34.0332 81.2328 33.9735 81.3388L30.4251 87.6388C30.1539 88.1204 29.459 88.1204 29.1878 87.6388L25.6394 81.3388C25.5797 81.2328 25.5484 81.1134 25.5484 80.9919V0.708986Z" fill="#243F5F"/>
+      <path d="M21.2903 25.7246V76.7012H12.7742V34.2207H0V25.7246H21.2903Z" fill="#243F5F"/>
+      <path d="M63.871 76.7012H72.3871V34.2207H76.6452V76.7012H85.1613V25.7246H63.871V76.7012Z" fill="#243F5F"/>
+      <path d="M38.3226 25.7246H59.6129V34.2207H46.8387V46.9649H59.6129V76.7012H38.3226V68.2051H51.0968V55.4609H38.3226V25.7246Z" fill="#243F5F"/>
+    </svg>
+    <h1>Notary Server {version}!</h1>
+    <ul>
+      <li>public key: <pre>{public_key}</pre></li>
+      <li>git commit hash: <a href="https://github.com/tlsnotary/tlsn/commit/{git_commit_hash}">{git_commit_hash}</a></li>
+      <li><a href="healthcheck">health check</a></li>
+      <li><a href="info">info</a></li>
+    </ul>
+  </body>
+
+concurrency: 32
+
+notarization:
+  max_sent_data: 4096
+  max_recv_data: 16384
+  timeout: 1800
+  private_key_path: null
+  signature_algorithm: secp256k1
+
+tls:
+  enabled: false
+  private_key_path: null
+  certificate_path: null
+
+log:
+  level: DEBUG
+  filter: null
+  format: COMPACT
+
+auth:
+  enabled: false
+  whitelist_path: null
+```
+⚠️ By default, `notarization.private_key_path` is `null`, which means a **random, ephemeral** signing key will be generated at runtime (see [Signing](#signing) for more details).
+
+### Overriding default
+The default setting can be overriden with either (1) environment variables, or (2) a configuration file (yaml).
+
+#### Environment Variables
+This will override each corresponding default value. It uses the prefix `NS_` followed by the configuration key(s) in uppercase. Double underscores are used for nested configuration keys, e.g. `tls.enabled` will be `NS_TLS__ENABLED`.
+
+Example:
+```bash
+NS_PORT=8080 NS_NOTARIZATION__MAX_SENT_DATA=2048 cargo run --release
+```
 
-### Using different setting files with Docker
-1. Instead of changing the key/cert/auth file path(s) in the config file, create a folder containing your key/cert/auth files by following the folder structure [here](./fixture/).
-2. When launching the docker container, mount your folder onto the docker container at the relevant path prefixed by `/root/.notary-server`.
-- Example 1: Using different key, cert, and auth files:
+#### Configuration File
+This will override all the default values, hence it needs to **contain all compulsory** configuration keys and values (refer to the [default yaml](#default)). This will take precedence over environment variables.
```

Comment:
```suggestion
This will override all the default values, hence it needs to **contain all compulsory** configuration keys and values (refer to the [default yaml](#default)). The config file has precedence over environment variables.
```